rts/motoko-rts/src/memory.rs

```diff
@@ -2,8 +2,29 @@
 pub mod ic;
 use crate::{constants::MAX_ARRAY_LENGTH_FOR_ITERATOR, types::*};
 
+use motoko_rts_macros::classical_persistence;
+use motoko_rts_macros::enhanced_orthogonal_persistence;
 use motoko_rts_macros::ic_mem_fn;
 
+#[cfg(feature = "ic")]
+use crate::constants::MB;
+
+#[enhanced_orthogonal_persistence]
+use crate::constants::GB;
+
+// TODO: Redesign for 64-bit support by using a dynamic partition list.
```

Comment:
Addressed in #4556, right?

Reply:
Yes, this PR would lift the restriction.